createdisk-library.sh

```diff
@@ -68,7 +68,7 @@ function create_qemu_image {
     fi
 
     sudo chown $USER:$USER -R $destDir
-    ${QEMU_IMG} rebase -b ${VM_PREFIX}-base $destDir/${VM_PREFIX}-master-0
+    ${QEMU_IMG} rebase -F qcow2 -b ${VM_PREFIX}-base $destDir/${VM_PREFIX}-master-0
```

Comment:
maybe we should also add `-f qcow2`?

Reply:
@cfergeau what's the difference?

Reply:
It just make sure the format is `qcow2`. I added it and it is make it clear.

Reply:
@gbraad `-F qcow2 -b ${VM_PREFIX}-base` go together, and `-f qcow2 $destDir/${VM_PREFIX}-master-0` go together. Each option specifies the format of a different argument.